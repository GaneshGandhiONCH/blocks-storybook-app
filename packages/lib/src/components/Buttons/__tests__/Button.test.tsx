describe("Button", () => {
  it("makes test pass", () => {
    // TODO
    expect(true).toBeTruthy();
  });
});
