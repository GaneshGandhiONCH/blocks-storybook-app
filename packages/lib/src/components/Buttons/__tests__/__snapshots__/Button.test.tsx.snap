// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button when state is disabled has a valid snapshot 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#14d64d",
          "borderRadius": 28,
          "height": 50,
          "justifyContent": "center",
          "paddingHorizontal": 20,
        },
        undefined,
      ]
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "white",
          "fontSize": 15,
        }
      }
    >
      ACCEPT
    </Text>
  </View>
</View>
`;

exports[`Button when state is loading has a valid snapshot 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#14d64d",
          "borderRadius": 28,
          "height": 50,
          "justifyContent": "center",
          "paddingHorizontal": 20,
        },
        undefined,
      ]
    }
  >
    <ActivityIndicator
      animating={true}
      color="#fff"
      hidesWhenStopped={true}
      size="small"
      style={
        Object {
          "alignSelf": "center",
          "position": "absolute",
        }
      }
    />
  </View>
</View>
`;

exports[`Button when state is normal has a valid snapshot 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#14d64d",
          "borderRadius": 28,
          "height": 50,
          "justifyContent": "center",
          "paddingHorizontal": 20,
        },
        undefined,
      ]
    }
  >
    <Text
      style={
        Object {
          "alignSelf": "center",
          "color": "white",
          "fontSize": 15,
        }
      }
    >
      ACCEPT
    </Text>
  </View>
</View>
`;
